README.md

```diff
@@ -1,120 +1,45 @@
+| ------- | ----------------------------- | ------------------------- |
+| GET     | api/v1/courses                | Fetches all courses       |
+| GET     | api/v1/courses/&lt;course_id> | Fetches a single course   |
+| PUT     | api/v1/courses/&lt;course_id> | Updates a single course   |
+| DELETE  | api/v1/courses/&lt;course_id> | Deletes a course          |
+| POST    | api/v1/auth/login             | Logs in a user            |
+| POST    | api/v1/auth/signup            | Registers a user          |
+| POST    | api/v1/auth/logout            | Logs out a user           |
+| POST    | api/v1/categories             | Creates a course category |
 
 ## Setup
-You should have **Python 3.5+** and **git** installed. 
-
-1. Clone the repo you've created from the template herein and change into the directory
-
-    ``
-    git clone <Your Repository>
-    ``
-
-2. Change into repo directory
-
-    ``
-    cd python-starter
-    ``
-
-3. Install poetry, a dependecy manager for python.
-
-    On windows, you will need powershell to install it:
-
-    ``
-    (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
-    ``
-
-    After that you will need to restart the shell to make it operational.
-
-    &nbsp;
-
-    On linux and other posix systems (mac included):
-
-    ``
-    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
-    ``
-
-    &nbsp;
-
-    To check that it is correctly installed, you can check the version:
-    ``
-    poetry --version
-    ``
-
-    May be the latest stable version is not installed with the installation script, to update poetry, you can run:
-
-    ``
-    poetry self update
-    ``
-
-4. With poetry installed, you should install project dependecies by running:
-
-    ``
-    poetry install
-    ``
-
-    This will install pytest for running tests and flake8, linter for your project.
-
-#### To Note
-`src/app.py` is the entry to the project and source code should go into the `src` folder.
-
-All tests should be written in the `tests` folder. tests/test_src.py is a sample test file that shows how tests should like. Feel free to delete it.
-
-#### Hints
-
-- Lint: `poetry run flake8`
-- Run tests using the command: `poetry run pytest`
-- Install dependencies: 
-  `poetry add <dependency>`
-- Install dev dependencies:
-  `poetry add --dev <dev-dependency>`
-- Run your project:
-  `poetry run app`
 
+1.  You should have **Python 3.5+** and **git** installed. 
+2.  Inorder to get started, clone this repository using `git clone https://github.com/BuildForSDG/Teachers-Hub-Backend.git`
+3.  Change in the repository directory `cd teachers-hub-backend`
+4.  Set up a virtual environment by running `python3 -m venv <name_of_the_environment>`
+5.  Install requirements by running `pip install -r requirements.txt`
+6.  Navigate to `http://localhost:8000` in the browser to view the api.
```

Comment:
You could include the command use to start the flask server as well before navigating to `http://localhost:8000`